### PR TITLE
Filtered out invoices in canceled state from bank reconcilation

### PR DESCRIPTION
**Pull request by tosink**

### CODE REVIEW

**Comment by tosink** [comment]
I have reviewed the errors on the 2 failing, it's some errors with your pre-commit and runboat. Nothing to do with the code. It's just one line of code added.

File: account_reconciliation_widget/models/reconciliation_widget.py [hunk 1]
```diff
@@ -103,6 +103,8 @@ def get_move_lines_for_bank_statement_line(
             mode=mode,
         )
 
+        domain.append(["move_id.state","!=","cancel"])
```
Comment (pedrobaeza) [inline account_reconciliation_widget/models/reconciliation_widget.py]: Respect pre-commit:

```suggestion
        domain.append(["move_id.state", "!=", "cancel"])
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Anyways, do this in `_domain_move_lines_for_reconciliation`, and only add `posted` entries (draft should be excluded as well).

**Comment by tosink** [comment]
I think I've passed all the checks now.

**Comment by tosink** [comment]
Any updates on the merging?

**Review by andrewdhastings — APPROVED** [review]
Looks good to me.

**Comment by andrewdhastings** [comment]
@pedrobaeza Is there anything preventing this from being merged?

**Comment by pedrobaeza** [comment]
Please squash commits into one.

**Comment by andrewdhastings** [comment]
@pedrobaeza Not sure if that worked? Not sure why the travis ci failed, I didn't change anything else.

**Comment by pedrobaeza** [comment]
The squash is not correct: you have now more commits than before.

**Comment by pedrobaeza** [comment]
I'm picking the diff myself, so please wait.

**Comment by andrewdhastings** [comment]
K, sorry, I can make a new branch and submit a fresh merge request if needs be. Sorry, not sure how to retroactively squash commits.

**Comment by pedrobaeza** [comment]
Pushed the fix in 9f85c3dc4f1492aae75559db45c4ac81efea2abf

**Comment by pedrobaeza** [comment]
Cherry-picked to v15 at f343ccfa3c09b3b206f49b2735b0f58068302b45

**Comment by andrewdhastings** [comment]
Perfect thanks! and thanks for adding to v15

**Pull request closed without merging**